pyomo/contrib/parmest/parmest.py

```diff
@@ -279,23 +728,32 @@ def __init__(
         assert isinstance(experiment_list, list)
         self.exp_list = experiment_list
 
-        # check that an experiment has experiment_outputs and unknown_parameters
-        model = self.exp_list[0].get_labeled_model()
+        # check if the experiment has a ``get_labeled_model`` function
         try:
-            outputs = [k.name for k, v in model.experiment_outputs.items()]
+            model = self.exp_list[0].get_labeled_model()
```

Comment:
This code seems to be repetitive with above. We probably want to modularize it. Maybe we can have a static method _get_labeled_model_helper( ) that takes the `self.exp_list[0]` in this case as an input and returns model. This helper function would contain the value error.